test/misc/bench-code.js

```diff
@@ -0,0 +1,6 @@
+/*1* KEEP THIS CODE AT THE TOP TO AVOID LINE NUMBER CHANGES */
```

Comment:
I was thinking about doing that.  I think that is something that I can do in a follow-up PR after landing the Typescript PRs to avoid merge conflicts.